examples/src/test/java/org/apache/ignite/testsuites/IgniteExamplesSelfTestSuite.java

```diff
@@ -101,6 +102,8 @@ public static TestSuite suite() throws Exception {
         // ML Grid.
         suite.addTest(IgniteExamplesMLTestSuite.suite());
 
+        suite.addTest(new TestSuite(EncryptedCacheExampleSelfTest.class));
```

Comment:
use suite.addTestSuite semantic.
see org.apache.ignite.testsuites.IgniteBasicTestSuite for example.